friture/octavefilters.py

```diff
@@ -48,12 +52,21 @@ def get_decs(self):
 
         return decs
 
+    def recalculate_calibration(self):
+        try:
+            
+            self.calibration = interp(self.fi, self.parent_widget.parent().dockmanager.parent().original_calibration_freqs, self.parent_widget.parent().dockmanager.parent().original_calibration)
```

Comment:
same comment as in audioproc.py: we would need to find a way to achieve this without having to walk through the parents